cpp/CMakeLists.txt

```diff
@@ -422,7 +414,7 @@ target_compile_definitions(cudf
 )
 
 # Disable Jitify log printing. See https://github.com/NVIDIA/jitify/issues/79
-target_compile_definitions(cudf PRIVATE "JITIFY_PRINT_LOG=0")
+target_compile_definitions(cudf PRIVATE "JITIFY_PRINT_LOG=1")
```

Comment:
Do we want this enabled by default?

Reply:
nope. thanks for pointing it out.

Reply:
will keep enabled for a bit and keep this comment open. still doing some testing locally.